Rock/Workflow/Action/Utility/InteractionAdd.cs

```diff
@@ -198,7 +198,7 @@ private InteractionComponentCache GetComponent( RockContext rockContext, Interac
                     int? interactionComponentId = new InteractionComponentService( rockContext )
                         .Queryable()
                         .AsNoTracking()
-                        .Where( c => c.EntityId.HasValue && c.EntityId.Value == entityId.Value )
+                        .Where( c => c.EntityId.HasValue && c.ChannelId == channel.Id && c.EntityId.Value == entityId.Value )
```

Comment:
I'm hesitant about this change because this is a core action. Unless this change has also been PR'd to core and is coming in an update at some point. If not, then the next time we pull it will want to overwrite this change.

Reply:
It has, and it has already been merged here: https://github.com/SparkDevNetwork/Rock/pull/4080/files

Reply:
So it looks like this is already in `alpha`? https://github.com/NewSpring/Rock/commit/55937d6da5e91b44ee1dffb240618762c53fe219

Reply:
It looks like the PR updated and this change isn't there anymore. Weird. But that's as it should be.